src/psyclone/psyir/transformations/hoist_trans.py

```diff
@@ -112,6 +113,10 @@ def apply(self, node, options=None):
         # Place the assignment node before the loop.
         loop.parent.children.insert(loop.position, node)
 
```

Comment:
Please could you update the class docstring to say that this transformation will also remove the parent loop if it no longer has a body.

Reply:
Good point. Done